src/releng_mapper/default.nix

```diff
@@ -33,6 +33,13 @@ let
       (fromRequirementsFile ./requirements-dev.txt python.packages);
     propagatedBuildInputs =
       (fromRequirementsFile ./requirements.txt python.packages);
+    dockerCmd = [
```

Comment:
@La0 to explain why I added you as reviewer.

If this works for releng_mapper, I think we should enable this for every backend project. What do you think?

Reply:
If that does not change the APIs implementation, sure

Reply:
nothing changes application wise.